### ISSUE TITLE
Crash when pressing trigger button

### ISSUE THREAD

**Issue by RichardMisiak** [quinton-ashley/cemu-no-gyro#5]
```
version: 1.0.15
controller: xbox one
platform: windows 10
```

steps to reproduce:
1. launch program with an xbox one controller connected
1. press one of the trigger buttons

The following error occurs:
```
Uncaught (in promise) ReferenceError: opt is not defined
    at loop (C:\Users\Richard\AppData\Local\Programs\cemu-no-gyro\resources\app.asar\views\js\index.js:123)
```
![image](https://user-images.githubusercontent.com/20317564/63228873-8f7b1c80-c1f1-11e9-9ccb-ec9a2652b631.png)

**Comment by quinton-ashley** [comment]
thank you for taking the time to open this issue.  Can't believe I didn't catch this mistake :( sorry

**Comment by quinton-ashley** [comment]
fixed in 1.0.16

**Comment by RichardMisiak** [comment]
Thanks for the quick fix! And thank for making this program, it's really useful
